container/openejb-jee/src/main/java/org/apache/openejb/jee/ObjectFactory.java

```diff
@@ -166,4 +167,12 @@ public JAXBElement<Webservices> createConnector(final Webservices value) {
     public JAXBElement<FacesConfig> createFacesConfig(final FacesConfig value) {
         return new JAXBElement<FacesConfig>(_FacesConfig_QNAME, FacesConfig.class, null, value);
     }
+    
+     /**
+     * Create an instance of {@link JAXBElement }{@code <}{@link WebFragment }{@code >}}
+     */
+    @XmlElementDecl(namespace = "http://java.sun.com/xml/ns/javaee", name = "web-fragment")
```

Comment:
We might need to add `web-fragment` part to `ObjectFactory$JAXB` in `openejb-jee-accessors` to fully support it?

Reply:
I added the web-fragment part to ObjectFactory$JAXB, but I did not solve the issue, because the org.apache.openejb.config.ReadDescriptors.mergeWebFragments calls JaxbJavaee.unmarshall which uses the NoSourceFilter.
It does not translate the namespace "https://jakarta.ee/xml/ns/jakartaee" to "http://java.sun.com/xml/ns/javaee".
So what would I have to do?
1. change  ReadDescriptors.mergeWebFragments to call JaxbJavaee.unmarshallJavaee which changes the namespace
2. change the NoSourceFilter to change the jakarta namespace as well not only the "http://xmlns.jcp.org/xml/ns/javaee"
3. anything else 

I think I might remove the web-fragment part from ObjectFactory$JAXB

Reply:
Thanks for the comment. I think, that the best way would be to follow up on the list. @dblevins might have some thoughts / insights here.